### PR TITLE
Update padding and grt adjustments for sky130 platforms

### PR DESCRIPTION
**Pull request by luis201420**

### CODE REVIEW

File: flow/platforms/sky130hd/fastroute.tcl [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-set_global_routing_layer_adjustment met4 0.4
-set_global_routing_layer_adjustment met3 0.4
-set_global_routing_layer_adjustment met2 0.4
-set_global_routing_layer_adjustment met1 0.4
+set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.3
+#set_global_routing_layer_adjustment met5 0.5
+#set_global_routing_layer_adjustment met4 0.4
+#set_global_routing_layer_adjustment met3 0.4
+#set_global_routing_layer_adjustment met2 0.4
+#set_global_routing_layer_adjustment met1 0.4
```
Comment (eder-matheus) [inline flow/platforms/sky130hd/fastroute.tcl]: You can remove these commented lines.

File: flow/platforms/sky130hs/fastroute.tcl [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-set_global_routing_layer_adjustment met4 0.4
-set_global_routing_layer_adjustment met3 0.4
-set_global_routing_layer_adjustment met2 0.4
-set_global_routing_layer_adjustment met1 0.4
+set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.3
+#set_global_routing_layer_adjustment met5 0.5
+#set_global_routing_layer_adjustment met4 0.4
+#set_global_routing_layer_adjustment met3 0.4
+#set_global_routing_layer_adjustment met2 0.4
+#set_global_routing_layer_adjustment met1 0.4
```
Comment (eder-matheus) [inline flow/platforms/sky130hs/fastroute.tcl]: Same here, remove the commented lines.

**Review by eder-matheus — CHANGES_REQUESTED** [review]

**Comment by eder-matheus** [comment]
@luis201420 These changes look good to me, but looking at the Google Sheet with the results of your experiments, it seems that running the GPL with timing_driven generates better results regarding time.
@maliberty do you think we should add it for the sky130 platforms?

**Comment by maliberty** [comment]
Are only jpeg & aes affected by the change?  I don't see updates to the other metrics.

**Comment by eder-matheus** [comment]
> Are only jpeg & aes affected by the change? I don't see updates to the other metrics.

We only changed these two designs because only they were failing. But yes, other designs are affected by the change. I can help Luis to update the metrics for them.

**Comment by maliberty** [comment]
@luis201420 @eder-matheus any update?

**Pull request closed without merging**